blockchain/blockchain-wallet-v4-frontend
# feat: deposit screen implemented wall-7

## Description (optional)
this PR implements deposit screen

## Testing Steps (optional)
Go over buy/sell flow, click on deposit and make sure that you see new deposit step
